libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -252,12 +253,27 @@ public void onCancelUploadForMediaDueToDeletedBlock(int mediaId) {
                         return otherMediaImageOptions;
                     }
 
+                    @Override
+                    public ArrayList<MediaOption> onGetOtherMediaFileOptions() {
+                        ArrayList<MediaOption> otherMediaDocumentOptions = initOtherMediaFileOptions();
```

Comment:
How about we keep these names the same? LMK :) 
```suggestion
                        ArrayList<MediaOption> otherMediaFileOptions = initOtherMediaFileOptions();
```

Reply:
Fixed.